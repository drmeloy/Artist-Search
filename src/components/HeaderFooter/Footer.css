@import url('https://fonts.googleapis.com/css?family=Montserrat:300&display=swap" rel="stylesheet');

.Footer {
    background-color: black;
    color: white; 
    margin-top: 80px;
    text-align: center;
    padding-right: 30px; 
    font-family: 'Montserrat', sans-serif;
}

.headerImg {
    width: 100%;
}