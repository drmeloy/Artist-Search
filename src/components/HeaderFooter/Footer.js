import React, { Component } from 'react';
import styles from './Footer.css';

export default class Footer extends Component {
  render() {
    return (
      <>
        <footer className={styles.Footer}>
          <img className={styles.headerImg} src="src/assets/header.png" /> 
          <p>&copy; 2020 -  Nick Brimmer ||  Dr. Mellow ||  Aaron Edward Glenn</p>
          
        </footer>
      </>
    );
  }
}
 