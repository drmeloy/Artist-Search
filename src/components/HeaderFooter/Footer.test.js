import React from 'react';
import { shallow }  from 'enzyme';
import Footer from './Footer';

describe('Footerz component', () => {
  it('renders a footer', () => {
    const wrapper = shallow(<Footer />); 
    expect(wrapper).toMatchSnapshot(); 
  });
});
