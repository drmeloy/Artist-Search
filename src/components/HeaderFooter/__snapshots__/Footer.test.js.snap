// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footerz component renders a footer 1`] = `
<Fragment>
  <footer
    className="Footer"
  >
    <p>
      Â© 2020 -  Nick Brimmer ||  Dr. Mellow ||  Aaron Edward Glenn
    </p>
  </footer>
</Fragment>
`;
